src/ui/components/Accordion.module.css

```diff
@@ -43,4 +43,7 @@
 
 .content.open {
   height: auto;
+  display: flex;
+  flex-direction: column;
+  overflow-y: hidden;
```

Comment:
this is probably the riskiest part, but it helps keep the outline filter input at the top